manifests/config/rsyslog.pp

```diff
@@ -46,7 +50,7 @@
 
     include 'logrotate'
 
-    $_rule_opts = $logrotate_options + {'log_files' => [ $log_file ]}
+    $_rule_opts = $logrotate_options + {'log_files' => [ $log_file ]} + {'create' => $logrotate_create}
```

Comment:
@kendall-moore You'll want `logrotate_create` to come before `logroate_options` if you don't want this to be a breaking change. Someone could have already set `create` in `logrotate_options` and this would override it.

Granted, it's probably to the same thing, but it might be nice to preserver whatever they did in case they set the mask differently for some reason.

Reply:
I see the problem @trevor-vaughan is describing, but giving `$logrotate_create` weaker precedence than a key inside `$logrotate_options` is counterintuitive behavior for a top-level class parameter―and inconsistent with the  `$log_file` parameter, which already (and unconditionally) overwrites its key in `$logrotate_options`.

Reply:
@op-ct That definitely makes sense, I just wanted folks to know that this could cause issues in currently running systems.